stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -316,6 +316,41 @@ extension Unsafe${Mutable}RawBufferPointer: ${Mutable}Collection {
     }
     return 0
   }
+
```

Comment:
You could include this in its own extension, as it is a customization point rather than something needed for `Collection` conformance.

Reply:
done